ThunderstoreCLI/Commands/InstallCommand.cs

```diff
@@ -33,13 +35,14 @@ public static async Task<int> Run(Config config)
         HttpClient http = new();
 
         int returnCode;
+        Match packageMatch;
         if (File.Exists(package))
         {
             returnCode = await InstallZip(config, http, def, profile, package, null);
         }
-        else if (FullPackageNameRegex.IsMatch(package))
+        else if ((packageMatch = FullPackageNameRegex().Match(package)).Success)
```

Comment:
Personally I'm not a fan of assigning values in places like inside the `if` condition since it often looks like a mistake and makes you wonder about the intention.